### DISCUSSION TITLE
future-ready talent self-learning modules do not get updated only show in progress but I completed them already.

### DISCUSSION THREAD

**Discussion by my-timepass** [github/india#833] [Students]
# Future-ready talent self-learning modules do not get updated only show in progress but I completed them already.
![Screenshot from 2022-04-26 12-24-52](https://user-images.githubusercontent.com/68778417/165240016-445dd4ae-e117-49ad-ae53-ea8adb28f639.png)

**Comment by prashantmore277** [comment]
Same Here, Any Solution for it?????
